september/src/main/webapp/WEB-INF/views/home.jsp

```diff
@@ -1,71 +1,131 @@
+	src="<c:url value="/resources/bootstrap/js/jquery.dataTables.js" />"></script>
+<!-- Data Table Bootstrap Plugin -->
+<script type="text/javascript"
+	src="<c:url value="/resources/bootstrap/js/dataTables.bootstrap.js" />"></script>
 <!-- Pagination Plugin -->
-
-<script type="text/javascript" src="<c:url value="/resources/bootstrap/js/jquery.simplePagination.js" />"></script>
+<script type="text/javascript"
+	src="<c:url value="/resources/bootstrap/js/jquery.simplePagination.js" />"></script>
 
 <script>
-jQuery(document).ready(function(){
-	 jQuery.each($(".date"), function(elem) {
-	        var date = new Date(elem.text());
-	        elem.text(date.toLocaleDateString());
-	    });
-/// Pagination plugin input values
+	var messageTable;
+	var filter;
+	function dtConstruction() {
+		messageTable = jQuery('#messageTable').dataTable({
+			"processing" : true,
+			"serverSide" : true,
+			"paging" : true,
+			"bFilter" : false,
+			"bLengthChange" : false,
+			"info" : true,
+			"dom" : '<"top"iflp<"clear">>rt<"bottom"iflp<"clear">>',
+			"deferRender" : true,
+			"ajax" : {
+				"url" : "datatable",
+				"contentType" : "application/json",
+				"data" : function(d) {
+					d.type = filter;
+				}
+			},
+			"dataSrc" : "",
+			"columns" : [ {
+				"data" : "applicationId"
+			}, {
+				"data" : "exceptionTimePrint"
+			}, {
+				"data" : "type"
+			}, {
+				"data" : "title"
+			}, {
+				"data" : "message"
+			} ]
+		});
+	}
+	jQuery(document).ready(function() {
+		console.log("Ready");
+		jQuery.each($(".date"), function(elem) {
+			var date = new Date(elem.text());
+			elem.text(date.toLocaleDateString());
+		});
 
-	 jQuery(".pagination").pagination({
-	        pages: ${totalPages},
-	        cssStyle: 'light-theme',
-	        hrefTextPrefix: '?page=',
-	        	currentPage: ${currentPageValue + 1}
-	    });
+		//Invoke the dataTable
+		filter = "";
+		dtConstruction();
 
-});
+	});
+
+	function filterme() {
+		//build a regex filter string with an or(|) condition
+		var types = jQuery('input:checkbox[name="type"]:checked').map(
+				function() {
+					return this.value;
+				}).get().join('|');
+		filter = types;
+		//reloads the dataTable with the new filtering options
+		messageTable.api().ajax.reload();
+	}
 </script>
 <sec:authorize access="isAuthenticated()">
 
-<nav aria-label="Page navigation" class="pull-right">
-<ul class="pagination"></ul>
-</nav>
+	<!-- Table display -->
+
+	<div class="row">
+		<div class="col-xs-12">
+			<b>Filter by Message Type:</b></br>
+			<form>
+				<div>
+					<input onchange="filterme()" type="checkbox" name="type"
+						value="ERROR|WARNING|INFO|DEBUG">All <input
+						onchange="filterme()" type="checkbox" name="type" value="ERROR">Error
+					<input onchange="filterme()" type="checkbox" name="type"
+						value="WARNING">Warning <input onchange="filterme()"
+						type="checkbox" name="type" value="DEBUG">Debug <input
+						onchange="filterme()" type="checkbox" name="type" value="INFO">Info
```

Comment:
fix formatting

Reply:
Will take care all the formatting and naming conventions